ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -19,6 +19,17 @@
 # This file defines fan-coil unit types. Valid types must include a fan and a coil of some type.
 # They must also include some control scheme (e.g. ZTC, DSP, etc.).
 
+FCU_RTM_DFVSC2X_CHWZTC_HWZTC_STM:
+  description: "FCU in a trench, with heating and cooling valves, and two fans."
+  is_canonical: true
+  implements:
+  - FCU
+  - RTM
+  - DFVSC2X
+  - CHWZTC
+  - HWZTC
+  - STM
```

Comment:
This should be DTM for consistency (use supply or discharge exclusively) -- discharge fans, discharge temp, etc.